sdk/search/Azure.AI.Search/tests/SessionRecords/SearchServiceClientTests/GetStastistics.json

```diff
@@ -0,0 +1,38 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://azs-net-epfichne.search.windows.net/servicestats?api-version=2019-05-06",
+      "RequestMethod": "GET",
+      "RequestHeaders": {
+        "api-key": "Sanitized",
+        "traceparent": "00-0ffaaab9de8c034aaa95887640a7b137-ae444107ba441f46-00",
+        "User-Agent": [
+          "azsdk-net-AI.Search/1.0.0-dev.20200227.1",
+          "(.NET Core 4.6.28325.01; Microsoft Windows 10.0.18363 )"
+        ],
+        "x-ms-client-request-id": "1c3b3b00f342ae0121d933ed0d823913",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": null,
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "Cache-Control": "no-cache",
+        "Content-Length": "637",
+        "Content-Type": "application/json; odata.metadata=minimal; odata.streaming=true",
+        "Date": "Thu, 27 Feb 2020 21:10:39 GMT",
+        "elapsed-time": "84",
+        "Expires": "-1",
+        "OData-Version": "4.0",
+        "Pragma": "no-cache",
+        "Preference-Applied": "odata.include-annotations=\u0022*\u0022",
+        "request-id": "1c3b3b00-f342-ae01-21d9-33ed0d823913",
+        "Strict-Transport-Security": "max-age=15724800; includeSubDomains"
+      },
+      "ResponseBody": "eyJAb2RhdGEuY29udGV4dCI6Imh0dHBzOi8vYXpzLW5ldC1lcGZpY2huZS5zZWFyY2gud2luZG93cy5uZXQvJG1ldGFkYXRhI01pY3Jvc29mdC5BenVyZS5TZWFyY2guVjIwMTlfMDVfMDYuU2VydmljZVN0YXRpc3RpY3MiLCJjb3VudGVycyI6eyJkb2N1bWVudENvdW50Ijp7InVzYWdlIjowLCJxdW90YSI6bnVsbH0sImluZGV4ZXNDb3VudCI6eyJ1c2FnZSI6MSwicXVvdGEiOjN9LCJpbmRleGVyc0NvdW50Ijp7InVzYWdlIjowLCJxdW90YSI6M30sImRhdGFTb3VyY2VzQ291bnQiOnsidXNhZ2UiOjAsInF1b3RhIjozfSwic3RvcmFnZVNpemUiOnsidXNhZ2UiOjAsInF1b3RhIjo1MjQyODgwMH0sInN5bm9ueW1NYXBzIjp7InVzYWdlIjowLCJxdW90YSI6M319LCJsaW1pdHMiOnsibWF4RmllbGRzUGVySW5kZXgiOjEwMDAsIm1heEluZGV4ZXJSdW5UaW1lIjoiUFQxTSIsIm1heEZpbGVFeHRyYWN0aW9uU2l6ZSI6MTY3NzcyMTYsIm1heEZpbGVDb250ZW50Q2hhcmFjdGVyc1RvRXh0cmFjdCI6MzI3NjgsIm1heEZpZWxkTmVzdGluZ0RlcHRoUGVySW5kZXgiOjEwLCJtYXhDb21wbGV4Q29sbGVjdGlvbkZpZWxkc1BlckluZGV4Ijo0MCwibWF4Q29tcGxleE9iamVjdHNJbkNvbGxlY3Rpb25zUGVyRG9jdW1lbnQiOjMwMDB9fQ=="
```

Comment:
We might need to change logic in ContentTypeUtilities to detect this response as JSON.

Reply:
I started looking into it for `text/plain` which was still dumped out as encoded bytes and I stopped there.  I can make a tweak to `Contains` for `.json` in `ContentTypeUtilities`.

Reply:
Can you file an issue. This might affect logging and exception messages too.

Reply:
Filed https://github.com/Azure/azure-sdk-for-net/issues/10216 to address it